app/api/scripts/create_docs.rb

```diff
@@ -1,9 +1,10 @@
 # frozen_string_literal: true
 
-require 'redcarpet'
+require 'commonmarker'
 
-renderer = Redcarpet::Render::HTML.new hard_wrap: true
-parser = Redcarpet::Markdown.new renderer
+def render_markdown(text)
+  CommonMarker.render_html(text, %i[UNSAFE HARDBREAKS], %i[autolink tagfilter])
```

Comment:
[Here's a CommonMark compliant implementation](https://github.com/codidact/qpixel/blob/d77ecafed5e892b8a6cee591f1f64f34b7ab0ba8/app/helpers/application_helper.rb#L58) - this won't quite be to spec. Try this:

```suggestion
CommonMarker.render_doc(text,
                        [:LIBERAL_HTML_TAG, :STRIKETHROUGH_DOUBLE_TILDE],
                        [:strikethrough, :autolink]).to_html(:UNSAFE)
```